src/main/kotlin/step3/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,28 @@
+package step3.racingcar
+
+object CarRacing {
+
+    fun execute(carCount: Int, moveCount: Int): ArrayList<Car> {
+        val carList = ArrayList<Car>()
+        repeat(carCount) {
+            carList.add(Car(executeCarMovements(moveCount)))
+        }
+        return carList
+    }
```

Comment:
```suggestion
    fun execute(carCount: Int, moveCount: Int): List<Car> = List(carCount) { Car(executeCarMovements(moveCount)) }
```

코틀린의 Collections API를 사용하고, 문(statement)를 식(expression)으로 만들면 이렇게 표현할 수 있습니다. 🤩

Reply:
오 꿀팁 감사합니다~